src/quic/node_quic_session.h

```diff
@@ -222,45 +222,47 @@ enum QuicSessionStatsIdx : int {
 
 class QuicSessionListener {
  public:
-  virtual ~QuicSessionListener();
+  virtual inline ~QuicSessionListener();
```

Comment:
So … `virtual inline` rarely makes sense, unfortunately. It only helps when the compiler can be *sure* of the actual subclass type for a given object in order to know which function to inline, but that doesn’t apply here, because we call these methods through a pointer that could ultimately refer to any given subclass, as far as I can tell – and in that case it reduces complexity a bit to keep it non-inline.